doc/source/tune/_tutorials/tune-60-seconds.rst

```diff
@@ -72,70 +69,78 @@ Use ``tune.run`` execute hyperparameter tuning using the core Ray APIs. This fun
 
 .. code-block:: bash
 
     == Status ==
     Memory usage on this node: 11.4/16.0 GiB
     Using FIFO scheduling algorithm.
-    Resources requested: 4/12 CPUs, 0/0 GPUs, 0.0/3.17 GiB heap, 0.0/1.07 GiB objects
+    Resources requested: 1/12 CPUs, 0/0 GPUs, 0.0/3.17 GiB heap, 0.0/1.07 GiB objects
     Result logdir: /Users/foo/ray_results/myexp
-    Number of trials: 4 (4 RUNNING)
+    Number of trials: 1 (1 RUNNING)
     +----------------------+----------+---------------------+-----------+--------+--------+----------------+-------+
     | Trial name           | status   | loc                 |    param1 | param2 |    acc | total time (s) |  iter |
     |----------------------+----------+---------------------+-----------+--------+--------+----------------+-------|
     | MyTrainable_a826033a | RUNNING  | 10.234.98.164:31115 | 0.303706  | 0.0761 | 0.1289 |        7.54952 |    15 |
-    | MyTrainable_a8263fc6 | RUNNING  | 10.234.98.164:31117 | 0.929276  | 0.158  | 0.4865 |        7.0501  |    14 |
-    | MyTrainable_a8267914 | RUNNING  | 10.234.98.164:31111 | 0.068426  | 0.0319 | 0.9585 |        7.0477  |    14 |
-    | MyTrainable_a826b7bc | RUNNING  | 10.234.98.164:31112 | 0.729127  | 0.0748 | 0.1797 |        7.05715 |    14 |
     +----------------------+----------+---------------------+-----------+--------+--------+----------------+-------+
 
+
+You can also easily run 10 trials (each trial is one instance of a Trainable). Tune automatically determines parallelism.
+
+.. code-block:: python
+
+    tune.run(trainable, num_samples=10)
+
+Finally, you can randomly sample hyperparameters:
+
+.. code-block:: python
+
+    space = {"x": tune.uniform(0, 1)}
+    tune.run(my_trainable, config=space, num_samples=10)
+
 See the documentation: :ref:`tune-run-ref`.
 
 
 Search Algorithms
 -----------------
 
-To optimize the hyperparameters of your training process, you will want to explore a “search space”.
-
-Search Algorithms are Tune modules that help explore a provided search space. It will use previous results from evaluating different hyperparameters to suggest better hyperparameters. Tune has SearchAlgorithms that integrate with many popular **optimization** libraries, such as `Nevergrad <https://github.com/facebookresearch/nevergrad>`_ and `Hyperopt <https://github.com/hyperopt/hyperopt/>`_.
+To optimize the hyperparameters of your training process, you will want to use a Search Algorithm which will help suggest better hyperparameters.
 
 .. code-block:: python
 
-    # https://github.com/hyperopt/hyperopt/
-    # pip install hyperopt
+    # Be sure to first run `pip install hyperopt`
+
     import hyperopt as hp
     from ray.tune.suggest.hyperopt import HyperOptSearch
 
     # Create a HyperOpt search space
-    space = {"momentum": hp.uniform("momentum", 0, 20), "lr": hp.uniform("lr", 0, 1)}
-    # Pass the search space into Tune's HyperOpt wrapper and maximize accuracy
+    space = {
```

Comment:
Can you add some comment.... either here or above this that you can swap this out for other suggest things too?

Reply:
one second doing this now